pennylane_qiskit/converter.py

```diff
@@ -179,7 +179,8 @@ def _function(params: dict = None, wires: list = None):
 
         # Processing the dictionary of parameters passed
         for op, qargs, cargs in qc.data:
-            instruction_name = op.__class__.__name__
+            # the new Singlton classes have different names than the objects they represent, but base_class.__name__ still matches
+            instruction_name = op.base_class.__name__
```

Comment:
was `op.base_class` introduced in a recent version? in other words, do we need to increase the qiskit lower-bound with this change?

Reply:
Oooh, good point. I'll fix it so it works both ways.